otj-metrics-core/src/main/java/com/opentable/metrics/ready/ReadyCheckRegistry.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
```

Comment:
This is a simplified stripped down version of CodaHales health registry, changed for health. We can't use much common code here, since the packages and concepts of HealthCheck and Result and the Registry are all embedded.